### PR TITLE
Alchemistry

### PR DESCRIPTION
**Pull request by CrankySupertoon**

### CODE REVIEW

File: docs/Mods/Alchemistry/ChemicalCombiner.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+# Package
+```zenscript
+mods.alchemistry.Combiner
+```
+
+## Methods
+- **[IItemStack](/Vanilla/Items/IItemStack/) output** - The output of the recipe.
+
+- **[IItemStack](/Vanilla/Items/IItemStack/) input** - The input of the recipe.
```
Comment (kindlich) [inline docs/Mods/Alchemistry/ChemicalCombiner.md:11]:
```suggestion
- **[IItemStack](/Vanilla/Items/IItemStack/)[] input** - The inputs of the recipe.
```

File: docs/Mods/Alchemistry/Dissolver.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Dissolver
+
+## Package
+`mods.alchemistry.Dissolver`
+
+## Methods
+- **[IItemStack](/Vanilla/Items/IItemStack/) input** - The input of the recipe.
+- **boolean relativeProbability** - Possibility Enable / Disable
+- **int rolls** -  Possibilty of Roles
+
```
Comment (kindlich) [inline docs/Mods/Alchemistry/Dissolver.md:10]: Please add a description for the probabilityGroups

File: docs/Mods/Alchemistry/Electrolyzer.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Electrolyzer
+
+# Package
+```zenscript
+`mods.alchemistry.Electrolyzer`
+```
+
+## Methods
+- **[ILiquidStack](/Vanilla/Liquids/ILiquidStack/) input** - The liquid input of the recipe.
+- **[IItemStack](/Vanilla/Items/IItemStack/) input** - The electrolyte input of the recipe.
```
Comment (kindlich) [inline docs/Mods/Alchemistry/Electrolyzer.md:10]:
```suggestion
- **[IItemStack](/Vanilla/Items/IItemStack/) electrolyte** - The electrolyte input of the recipe.
```

File: docs/Mods/Alchemistry/Liquifier.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# Liquifier
+
+# Package
+```zenscript
+mods.alchemistry.Liquifier
+```
+
+## Methods
+**[IItemStack](/Vanilla/Items/IItemStack/) output** - The output of the recipe.
```
Comment (kindlich) [inline docs/Mods/Alchemistry/Liquifier.md:9]: Is a liquid stack

**Comment by CrankySupertoon** [comment]
hmm why did this close...

**Comment by CrankySupertoon** [comment]
wonder if my force push broke it

**Comment by CrankySupertoon** [comment]
#280

**Pull request closed without merging**